egs/wsj/s5/steps/nnet3/chain/build_tree.sh

```diff
@@ -25,6 +25,7 @@ frame_subsampling_factor=1
 leftmost_questions_truncate=10
 tree_stats_opts=
 cluster_phones_opts=
+repeat_frames=false
```

Comment:
Should I limit line length (to 80 chars) here too? Because it will get ugly if I do.